src/components/comment-modifiers/comment-modifiers.jsx

```diff
@@ -0,0 +1,60 @@
+import * as api from "../../utils/api";
+import "./comment-modifiers.css";
+import { useState } from "react";
+
+const CommentModifiers = ({ comment }) => {
+  const [disableUp, setDisableUp] = useState(false);
+  const [disableDown, setDisableDown] = useState(false);
+
+  const [singleComment, setSingleComment] = useState([]);
+
+  const deleteComment = () => {
+    api.deleteComment(comment.comment_id).then((res) => console.log(res))}
+
+  const handleCommentVotes = (num) => {
+    const voteChangeOnComment = { inc_votes: num };
+    api
+      .updateVotesOnComment(comment.comment_id, voteChangeOnComment)
+      .then((res) => console.log(res + "UPDATED VOTE"));
+
+    setSingleComment((currentSingleComment) => {
+      return {
+        ...currentSingleComment,
+        votes: currentSingleComment.votes + num,

```

Comment:
I would re-order this to be above the api request for some optimistic rendering. Will also need to account for if the server is down/internet, should optimistically render, then reset if the api request goes wrong. 

The UI is really slow at the moment